vendor/modules.txt

```diff
@@ -1214,13 +1214,17 @@ open-cluster-management.io/api/client/cluster/clientset/versioned/typed/cluster/
 open-cluster-management.io/api/client/cluster/clientset/versioned/typed/cluster/v1/fake
 open-cluster-management.io/api/client/cluster/clientset/versioned/typed/cluster/v1alpha1
```

Comment:
@skitt  Should I remove these v1alpha1 references manually?

Reply:
No, the whole `vendor/modules.txt` is automatically managed by `go mod vendor`; if the entries are still present, something (a dependency) is still using them.

Reply:
In fact it’s the OCM API itself which uses them, since it still supports the v1alpha1 API.